### ISSUE TITLE
Android Gradients support

### ISSUE THREAD

**Issue by Muhib** [LeaVerou/prefixfree#116]
I added gradients in one of my pages and tested in Android. The script should have worked in Android Browser, 2.3.6 Gingerbread. But it's not working. Looks like it's a bug of yours.

Actually, I started using your script so that it supports gradients in android (actually, I can't find any possible solution about the vendor prefix of android's gradients. So I tried your script if it works). 

So, please support Android's Default Browser's (2.3.6 Gingerbread especially) gradients.

**Comment by oscar-broman** [comment]
http://stackoverflow.com/questions/8882086/css-gradient-not-working-on-android-2-3-6

Put the outdated syntax **below** the prefix free gradient, with the `-webkit-` prefix.

**Comment by LeaVerou** [comment]
Nope, you need to put it above, so that it's overridden with the new one for browsers that support both.
This tool might be helpful, although it's a bit outdated (but then again, so is `-webkit-gradient()`: http://lea.verou.me/demos/cssgradientsplease/
